source_pubsub/client.go

```diff
@@ -52,6 +53,7 @@ type PubSubClient struct {
 	conn         *grpc.ClientConn
 	pubSubClient proto.PubSubClient
 	subClient    proto.PubSub_SubscribeClient
+	mu           *sync.Mutex
```

Comment:
What is this used for?

Reply:
Woops accidently left over, we dont need it